libc/src/__support/OSUtil/linux/x86_64/vdso.h

```diff
@@ -0,0 +1,56 @@
+//===---------- x86/x86_64 vdso configuration ---------------------* C++ *-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+#ifndef LLVM_LIBC_SRC___SUPPORT_OSUTIL_LINUX_X86_64_VDSO_H
+#define LLVM_LIBC_SRC___SUPPORT_OSUTIL_LINUX_X86_64_VDSO_H
+#include "src/__support/CPP/string_view.h"
+namespace LIBC_NAMESPACE {
+namespace vdso {
+// macro definitions
+// following the order in
+// https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/arch/x86/entry/vdso/vdso.lds.S
```

Comment:
Reading the kernel sources, there's an additional optional symbol: __vdso_sgx_enter_enclave.

Also, there's `__vdso_` prefixed symbols. Does that change the logic needed for finding the right symbols for x86?

Reply:
The manpage suggest we should just ignore those without `__vdso` prefix.

Reply:
I will add a comment for this